### PR TITLE
some linting

### PR DESCRIPTION
**Pull request by fperrad**
after merge of #361

### CODE REVIEW

File: tommath.h [hunk 1]
```diff
@@ -385,8 +385,8 @@ mp_err mp_unpack(mp_int *rop, size_t count, mp_order order, size_t size, mp_endi
 
 /* pack binary data */
 size_t mp_pack_count(const mp_int *a, size_t nails, size_t size) MP_WUR;
-mp_err mp_pack(void *rop, size_t maxcount, size_t *writtencount, mp_order order, size_t size, mp_endian endian,
-               size_t nails, const mp_int *op) MP_WUR;
+mp_err mp_pack(void *rop, size_t maxcount, size_t *written, mp_order order, size_t size,
```
Comment (sjaeckel) [inline tommath.h]: @minad `written` or `writtencount`?

Comment (fperrad) [reply tommath.h]: `written` is used by `mp_to_ubin`, `mp_to_sbin` and `mp_to_radix`.

Comment (sjaeckel) [reply tommath.h]: yes, but `written` there is the number of bytes written, whereas `written` here is the number of elements of size `size`

Comment (minad) [reply tommath.h]: written is ok I guess

**Pull request merged**